### PR TITLE
Add markdown preview to SubmissionGradingEditor

### PR DESCRIPTION
**Pull request by kryonix**
Add real time Markdown rendering to student feedback editor for tutors.

### CODE REVIEW

File: src/Components/CommonElements.elm [hunk 1]
```diff
@@ -37,6 +37,7 @@ module Components.CommonElements exposing
     , timeInputElement
     , viewFormErrors
     , widePage
+    , widePage2
```
Comment (PatWie) [inline src/Components/CommonElements.elm]: Is there a more descriptive name instead of appending a 2?

Comment (kryonix) [reply src/Components/CommonElements.elm]: Good question. The only difference between `widePage` and `widePage2` is, that `TC.mw8_l` has been removed from the classes list, such that `TC.mw9_ns` works. 

Removing this entry from `widePage` broke the layout at several places.

Comment (vork) [reply src/Components/CommonElements.elm]: So in theory we could basically create a `mw-8` and a `mw-9` version of `widePage`. The `TC.mw9_ns` is indeed not working on `widePage` and can be removed. The `TC.w_100`  should anyway keep the width du full frame. If I see it correctly `widePage-mw8` should be fine with just a `TC.mw8_ns` and `widePage-mw9` should be just fine with `TC.mw9_ns` and `TC.w100` then handles the small case.

Comment (kryonix) [reply src/Components/CommonElements.elm]: I did rename `widePage` to `widePage_mw8` and `widePage2` to `widePage_mw9`. These names should be more descriptive.

**Review by vork — APPROVED** [review]
Nice. Seems good to me now.

**Pull request merged**